website/docs/r/search_index.html.markdown

```diff
@@ -258,10 +258,10 @@ An [Atlas Search analyzer](https://docs.atlas.mongodb.com/reference/atlas-search
 
 * `token_filters` - Array containing zero or more token filters. Always require a type field, and some take additional options as well:
   ```terraform
-  "tokenFilters":{
+  "tokenFilters":[{
```

Comment:
I guess this PR is gonna be a breaking change?

Reply:
we agreed with Zuhair that it's not a breaking change but it's really a documentation bug.  In fact the title is fine "Array containing zero or more token filters" but the example is wrong